apps/costcontrol/test/unit/startup_test.js

```diff
@@ -450,4 +450,68 @@ suite('Application Startup Modes Test Suite >', function() {
       CostControlApp.init();
     }
   );
+
+  test(
+    'DSDS Ensure the FTE will be closed the second time when there is a data' +
```

Comment:
As we discovered the bug was for `updateUI` to not be calling the `callback`, the regression tests can be simplified to just test that function always calls the callback. Please, test only `updateUI()`.